frontend-react/src/utils/Faker.ts

```diff
@@ -0,0 +1,161 @@
+     *
+     * @param numberOrRange The number or range to convert.
+     * @param numberOrRange.min The minimum value for the range.
+     * @param numberOrRange.max The maximum value for the range.
+     *
+     * @example
+     * faker.helpers.rangeToNumber(1) // 1
+     * faker.helpers.rangeToNumber({ min: 1, max: 10 }) // 5
+     *
+     * @since 8.0.0
+     */
+    rangeToNumber(
+        numberOrRange:
+            | number
+            | {
+                  /**
+                   * The minimum value for the range.
+                   */
+                  min: number;
+                  /**
+                   * The maximum value for the range.
+                   */
+                  max: number;
+              }
+    ): number {
+        if (typeof numberOrRange === "number") {
+            return numberOrRange;
+        }
+
+        return this.datatype.number(numberOrRange);
+    }
+
+    /**
+     * Generates an array containing values returned by the given method.
+     *
+     * @template T The type of elements.
+     * @param method The method used to generate the values.
+     * @param options The optional options object.
+     * @param options.count The number or range of elements to generate. Defaults to `3`.
+     *
+     * @example
+     * faker.helpers.multiple(faker.person.firstName) // [ 'Aniya', 'Norval', 'Dallin' ]
+     * faker.helpers.multiple(faker.person.firstName, { count: 3 }) // [ 'Santos', 'Lavinia', 'Lavinia' ]
+     *
+     * @since 8.0.0
+     */
+    multiple<T>(
+        method: () => T,
+        options: {
+            /**
+             * The number or range of elements to generate.
+             *
+             * @default 3
+             */
+            count?:
+                | number
+                | {
+                      /**
+                       * The minimum value for the range.
+                       */
+                      min: number;
+                      /**
+                       * The maximum value for the range.
+                       */
+                      max: number;
+                  };
+        } = {}
+    ): T[] {
+        const count = this.rangeToNumber(options.count ?? 3);
+        if (count <= 0) {
+            return [];
+        }
+
+        return Array.from({ length: count }, method);
+    }
+
+    // eslint-disable-next-line @typescript-eslint/no-useless-constructor
+    constructor(faker: OrigFaker) {
+        super(faker);
+    }
+}
+
+/**
+ * Not sure how to tell typescript that our class extends HelpersModule
+ * naturally due to using prototype constructor. Remove this type and
+ * use HelpersModuleBase directly when we figure out how or reach v8.0.
+ */
+export type HelpersModule = OrigHelpersModule & {
+    rangeToNumber: HelpersModuleBase["rangeToNumber"];
+    multiple: HelpersModuleBase["multiple"];
+};
+
+/**
+ * Hybrid Faker with some safe features from 8.0 alpha. Allows storing
```

Comment:
Can we just specify the v8 alpha the version for Faker in our codebase?  I don't think we should maintain this logic internally, and if there are any changes before v8 goes live, we should adjust anyway so we don't have long-lived deviations from the library.